webui/react/src/services/apiConfig.ts

```diff
@@ -24,6 +24,7 @@ const generateApiConfig = (apiConfig?: Api.ConfigurationParameters) => {
   const config = updatedApiConfigParams(apiConfig);
   return {
     Auth: new Api.AuthenticationApi(config),
+    CheckPoint: Api.CheckpointsApiFetchParamCreator(config),
```

Comment:
my opinion is for consistency Checkpoint should not have uppercase P (here or in `useModalCheckpoint.tsx`)